_documentation/user/flags/wings.md

```diff
@@ -5,4 +5,25 @@ type: Good
 summary: Tank can drive in air.
 ---
 
-...
+Wings flag allows the player to maneuver their tank while in the air, enabling them to move forward, backward, or turn to either side (x and y axis), all while the gravity still affects the tank midair (z axis).
+
+## Potency
+
+The server variables affecting Wings can greatly change the potency of the flag, but in general, it is considered to be one of the most powerful flags in the game.
```

Comment:
This is thin. Opinion is not a valid description of potency. The ability to maneuver in the air makes it more difficult to predict where an opponent will land. The ability to move higher than a single jump makes it possible to easily reach heights otherwise impossible or difficult to get to. et cetera.

Reply:
> move higher than a single jump

This brings up a good point that the number of jumps is determined by a BZDB variable. By default in 2.4, Wings is configured to be one jump only.